src/OpenTK/Input/JoystickDevice.cs

```diff
@@ -263,17 +263,6 @@ internal JoystickAxisCollection(int numAxes)
         }
 
         /// <summary>
-        /// Gets a System.Single indicating the absolute position of the JoystickAxis.
```

Comment:
It (was) now a duplicate function.
https://github.com/leezer3/opentk/blob/5c11f61729f56efbb34f5882fd9932f2cc30cf06/src/OpenTK/Input/JoystickDevice.cs#L259